src/browsers/schema/content/results.md

```diff
@@ -0,0 +1,8 @@
+Hover:
```

Comment:
Tooltips added in ccae47fa2ed23f602865df438978ac3774b4d477. This file can be deleted.

Reply:
Deleted results.md